### DISCUSSION TITLE
Correct fcos version for OKD 4.11

### DISCUSSION THREAD

**Discussion by dkoci** [okd-project/okd#1453] [General]
Which is the correct FCOS version for OKD 4.11.
Command `openshift-install coreos print-stream-json` is not working anymore.

**Comment by nate-duke** [comment]
https://amd64.origin.releases.ci.openshift.org/releasestream/4-stable/release/4.11.0-0.okd-2022-12-02-145640

Says: Fedora CoreOS 411.36.4

**Comment by vrutkovs** [comment]
https://github.com/okd-project/okd/blob/master/FAQ.md#which-fedora-coreos-should-i-use

**Reply by dkoci** [reply]
`openshift-install coreos print-stream-json` is not working anymore. It was working on 4.8 version.

**Reply by vrutkovs** [reply]
What does it print? What's the output of `openshift-install version`?

**Comment by JaimeMagiera** [comment]
The latest 4.11 stable definitely works. 

`[core@ip-10-0-1-186 ~]$ ./openshift-install version
./openshift-install 4.11.0-0.okd-2022-12-02-145640
built from commit c258fd3a0027ef3fa4ed685e093e044b1d4014cd
release image quay.io/openshift/okd@sha256:924958ed5ba93ea9d72e4b5a319d0d4df220edbb891816c106215cbf318a0a40
release architecture amd64`


`[core@ip-10-0-1-186 ~]$ ./openshift-install coreos print-stream-json | less
{
    "stream": "stable",
    "metadata": {
        "last-modified": "2022-07-26T22:48:18Z",
        "generator": "fedora-coreos-stream-generator v0.2.7"
    },
    "architectures": {
        "aarch64": {
            "artifacts": {
                "aws": {
                    "release": "36.20220716.3.1",
                    "formats": {
                        "vmdk.xz": {
                            "disk": {
                                "location": "https://builds.coreos.fedoraproject.org/prod/streams/stable/builds/36.20220716.3.1/aarch64/fedora-coreos-36.20220716.3.1-aws.aarch64.vmdk.xz",
                                "signature": "https://builds.coreos.fedoraproject.org/prod/streams/stable/builds/36.20220716.3.1/aarch64/fedora-coreos-36.20220716.3.1-aws.aarch64.vmdk.xz.sig",
                                "sha256": "50114b48c259917cc23e761e840ead7b198de329885ae0f21f7221debab5826a",
                                "uncompressed-sha256": "a0221aca5415844ef67cee7a41130c4be23af82a62ed32ea045be623a20466d5"
                            }
                        }
                    }
                },
`
